packages/pwa-kit-extension-sdk/src/configs/babel/utils.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2024, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import * as fse from 'fs-extra'
+import * as p from 'path'
+import {getApplicationExtensionInfo} from '../../shared/utils/extensibility'
+
+/**
+ * Builds Babel extensibility arguments for processing specific files and paths.
+ * Accessing files via "node_modules" ensures compatibility in both mono-repos and
+ * developers' environments working on templates with extensibility. Avoids relative paths
+ * or symlinked paths that Babel doesn't support by using realpathSync.
+ */
+export const buildBabelExtensibilityArgs = () => {
+    const {installed}: {installed: (string | undefined)[]} = getApplicationExtensionInfo()
+
+    // Filter out undefined values from installed extensions
+    const filteredInstalled = installed.filter(
+        (packageName): packageName is string => packageName !== undefined
+    )
+
+    const serverPath = fse.realpathSync(
```

Comment:
Did you test that this solution works on a generated project? Although I did write the code to be agnostic of being in the monorepo or not it would be a good idea to have a second person confirm that.

Reply:
Yes it works on a generated project. I generated an PWA Kit Application using the `extension-sample` and confirmed there are no Babel warnings when running `npm start`.